### PR TITLE
test: leverage e2e

### PR DESCRIPTION
**Pull request by toteki**
Simple leverage e2e additions

update: Also includes a fix in GetAccountPosition

### CODE REVIEW

File: tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -18,6 +20,36 @@ func TestE2ETestSuite(t *testing.T) {
 	suite.Run(t, new(E2ETest))
 }
 
+// mustEventuallySucceedTx executes an sdk.Msg (retrying several times if receiving any error,
```
Comment (kosegor) [inline tests/e2e/e2e_test.go:23]:
```suggestion
// mustEventuallySucceedTx executes an sdk.Msg, retrying several times if receiving any error,
```

**Review by kosegor — APPROVED** [review]

**Comment by gsk967** [comment]
We used validators accounts for signing because those accounts have more voting power when submit the votes to gov proposals @toteki

**Review by gsk967 — APPROVED** [review]

File: tests/e2e/e2e_test.go [hunk 2]
```diff
@@ -18,6 +20,36 @@ func TestE2ETestSuite(t *testing.T) {
...
+// mustEventuallySucceedTx executes an sdk.Msg, retrying several times if receiving any error,
+// and requires that the transaction eventually succeeded with nil error. Since this function
+// retries for 30 seconds and ignores errors, it is suitable for scenario setup transaction or
+// those which might require time elapsing before they succeed.
+func (s *E2ETest) mustEventuallySucceedTx(msg sdk.Msg) {
+	s.Require().Eventually(
+		func() bool {
+			return s.BroadcastTxWithRetry(msg) == nil
+		},
+		30*time.Second,
```
[the remark this answers is not in the document]
Comment (toteki) [reply tests/e2e/e2e_test.go]: changed to 5sec

File: x/leverage/keeper/position.go [hunk 3]
```diff
@@ -21,7 +21,7 @@ var minimumBorrowFactor = sdk.MustNewDecFromStr("0.5")
 func (k Keeper) GetAccountPosition(ctx sdk.Context, addr sdk.AccAddress, isForLiquidation bool,
 ) (types.AccountPosition, error) {
 	tokenSettings := k.GetAllRegisteredTokens(ctx)
-	specialPairs := []types.SpecialAssetPair{}
+	specialPairs := k.GetAllSpecialAssetPairs(ctx)
```
Comment (toteki) [inline x/leverage/keeper/position.go:24]: This is a fix

**Pull request merged**